docs/docsite/rst/user_guide/playbooks_reuse_include_tasks_example.rst

```diff
@@ -0,0 +1,153 @@
+              - htop
+              - atop
+              - okular
+              - gcc
+        - name: remove RHEL software
+          yum:
+            name: '{{ packages }}'
+            state: absent
+          vars:
+            packages:
+              - neovim
+              - gnome-weather
+              - orca
+              - evince
+              - mg
+              - nano
+              - pico
+              - gnome-boxes
+
+Dismantling The Initial Playbook
+````````````````````````````````
+Original_playbook.yml is not overly large, but given time, it could get big and ugly. Let's break it down into task files. Each task file below, represents one of the actions that original_playbook.yml performed - update server, install repos, install software and remove software. The task files are named as such:
+
+- update_my_box.yml - a task file for updating a server
+- install_github_repos.yml - a task file for installing Github repositories
+- install_RHEL_software.yml - a task file for installing some useful desktop software
+- remove_RHEL_software.yml - a task file for removing software that is no longer needed
+
+Their contents are below::
+
+        ---
+        # update_my_box.yml
+        - name: update server
+          yum:
+            name: '*'
+            state: latest
+
+        ---
+        # install_github_repos.yml
+        - name: add epel-release
+          yum:
+            name: epel-release
+            state: present
+        - name: add rpmfusion
+          yum:
+            name: https://download1.rpmfusion.org/free/el/rpmfusion-free-release-7.noarch.rpm
+            state: present
+
+        ---
+        # install_RHEL_software.yml
+        - name: install RHEL software
+          yum:
+            name: '{{ packages }}'
+            state: present
+          vars:
+            packages:
+              - hamster-time-tracker
+              - kate
+              - Zim
+              - vlc
+              - htop
+              - atop
+              - okular
+              - gcc
+
+        ---
+        # remove_RHEL_software.yml
+        - name: remove RHEL software
+          yum:
+            name: '{{ packages }}'
+            state: absent
+          vars:
+            packages:
+              - neovim
+              - gnome-weather
+              - orca
+              - evince
+              - mg
+              - nano
+              - pico
+              - gnome-boxes
+
+Recreating The New Playbook
+```````````````````````````
+The file Original_playbook.yml has been split into four small, (reusable), task books. The include_tasks module can be used to create a new playbook (named RHEL_desktop.yml) that calls those task books, as shown below:: 
+
+        ---
+        # RHEL_desktop.yml
+        # building a RHEL desktop
+        - hosts: all
+          remote_user: root
+          tasks:
+          - name: update my box
+            include_tasks: /home/redfern/ansible/playbooks/update_my_box.yml
```

Comment:
If you're doing an unconditional include_tasks, you might consider instead to use import_tasks.

Reply:
I will think further about this. There may be more I can write, so that "include_tasks" and "import_tasks" are both covered.